api/sentry/api_sentry.py

```diff
@@ -81,31 +117,33 @@ def sentry_sessions_crash_free_rate(self, crash_free_rate_type, release):
     def sentry_adoption_rate(self, release):
         health_info_release = self.client.http_get(
             (
-                "organizations/{0}/releases/org.mozilla.ios.Firefox%40{1}/"
-                "?health=1&summaryStatsPeriod=7d&project={2}"
-                "&environment=Production&adoptionStages=1"
-            ).format(self.organization_slug, release, self.project_id)
+                "organizations/{0}/releases/{1}%40{2}/"
+                "?health=1&summaryStatsPeriod=7d&project={3}"
+                "&environment={4}&adoptionStages=1"
+            ).format(
+                self.organization_slug, self.package,
+                release, self.project_id, self.environment)
         )
         # Long version name could be beta
-        if health_info_release is None:
-            return self.client.http_get(
-                (
-                    "organizations/{0}/releases/"
-                    "org.mozilla.ios.FirefoxBeta%40{1}/"
-                    "?health=1&summaryStatsPeriod=7d&project={2}"
-                    "&environment=Production&adoptionStages=1"
-                ).format(self.organization_slug, release, self.project_id)
-            )
-        else:
-            return health_info_release
+        # if health_info_release is None:
```

Comment:
Should we keep this or can we delete these lines?

Reply:
We can delete these lines.